webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministration/UsersAdministration.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { observer } from 'mobx-react-lite';
+
+import type { AdministrationItemContentComponent } from '@cloudbeaver/core-administration';
+import { s, SContext, StyleRegistry, ToolsPanel, useS, useTranslate } from '@cloudbeaver/core-blocks';
+import { useService } from '@cloudbeaver/core-di';
+import { baseTabStyles, ITabData, Tab, TabList, TabPanel, tabPanelStyles, TabsState, TabTitle, underlineTabStyles } from '@cloudbeaver/core-ui';
+
+import { TeamsPage } from '../Teams/TeamsPage';
+import { EUsersAdministrationSub, UsersAdministrationNavigationService } from '../UsersAdministrationNavigationService';
+import { UsersPage } from '../UsersTable/UsersPage';
+import style from './styles/UsersAdministration.m.css';
+import tabStyle from './styles/UsersAdministrationTab.m.css';
+import tabPanelStyle from './styles/UsersAdministrationTabPanel.m.css';
+
+const tabPanelRegistry: StyleRegistry = [[tabPanelStyles, { mode: 'append', styles: [tabPanelStyle] }]];
+
+const mainTabsRegistry: StyleRegistry = [[baseTabStyles, { mode: 'append', styles: [underlineTabStyles, tabStyle] }]];
+
+export const UsersAdministration: AdministrationItemContentComponent = observer(function UsersAdministration({ sub, param }) {
```

Comment:
This file was moved, but GitHub detected it as a delete + new file. Please fix this. It's very important to save the commit history

Reply:
done